README.md

```diff
@@ -6,22 +6,140 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
+
+-   **down** - Service is down and it is not possible to process the request.
+-   **problem** - Service is running, but some problem occured during processing the request.
+-   **running** - Service is running and the request was processed successfully.
+
+### Cronjob healthcheck
+
+Unlike services healthcheck, the cronjobs healthcheck are not refreshed at one minute intervals. They simply show the status of the last cronjob run. Most of them are performed once a day, so there is no need for refresh. The status of each cronjob can acquire following 2 states:
+
+-   **Fail** - Error occurred while running the script and the script did not finished successfully.
+-   **Success** - The script ran successfully, without errors.
+
+### Usage
+
+This tab has mostly informative value, the background colors of individual headings corresponds to the status of service/cronjob runs (the same colors are used as for the traffic light). However, it is possible to click on reload button at services, to send request to the service manually. Also, if an error occurs while running the cron job, a small icon with letter "i" is displayed next to the name of job. After clicking on the icon, an error message will be displayed in the modal window.
+
+## Logs
+
+The logs tab is used to display and filter log messages from selected log files. Log messages are displayed from the newest to the oldest.
+It is possible to display log messages from several log files at once. In this case the logs from first log file are displayed, followed by logs from second file etc.
+
+### Usage
+
+The filter provides several properties according to which log messages can be filtered. Each of this properties has default value in the case that user does not set its value. After clicking **Filter** button, log messages are displayed in table under the filter.
+
+## Users
+
+It is possible to perform CRUD operation over MySQL tables, where information about users are stored. Currently there are two tables available:
+
+-   **approved users**
+-   **users waiting for approval**
+
+### Usage
+
+The users tab provides CRUD functionality over both tables. Also there is extra approval functionality for users, that are stored in **users waiting for approval** table. After clicking on green check icon, modal window with the form is displayed. To successfully validate the user, it is necessary to fill in either SDO or Vendor access rights.
+
+## Config
+
+The config tab simply displays the content of the configuration file, which contains the individual configuration details of the yangcatalog.
+
+### Usage
+
+Users with the permissions granted have the option to update the content of the configuration file by clicking on the **Edit** button. After applying the changes, it is necessary to click the **Save** button for saving changes.
```

Comment:
At this at the end of the usage.

The save will automatically reload configuration of each service (backend, yang-search, yangvalidator...) so it should take effect immediately and next usage of these services should use the newly set variables.